src/data/__tests__/networkConfig.test.ts

```diff
@@ -0,0 +1,66 @@
+import { SUPPORTED_NETWORK } from "@/data/network.config";
+import { Network } from "@/data/types";
+
+describe("SUPPORTED_NETWORK", () => {
+  it("contains Ethereum", () => {
+    const ethereum: Network | undefined = SUPPORTED_NETWORK["ethereum"];
+    expect(ethereum).toBeDefined();
+    expect(ethereum?.displayName).toBe("Ethereum");
+  });
+
+  it("contains Binance Smart Chain", () => {
+    const bsc: Network | undefined = SUPPORTED_NETWORK["bsc"];
+    expect(bsc).toBeDefined();
+    expect(bsc?.displayName).toBe("Binance Smart Chain");
+  });
+
+  it("contains Polygon", () => {
+    const polygon: Network | undefined = SUPPORTED_NETWORK["polygon"];
+    expect(polygon).toBeDefined();
+    expect(polygon?.displayName).toBe("Polygon");
+  });
+
+  it("contains Optimism", () => {
+    const optimism: Network | undefined = SUPPORTED_NETWORK["optimism"];
+    expect(optimism).toBeDefined();
+    expect(optimism?.displayName).toBe("Optimism");
+  });
+
+  it("contains Arbitrum", () => {
+    const arbitrum: Network | undefined = SUPPORTED_NETWORK["arbitrum"];
+    expect(arbitrum).toBeDefined();
+    expect(arbitrum?.displayName).toBe("Arbitrum");
+  });
+
+  it("contains Avalanche C-Chain", () => {
+    const avalancheCChain: Network | undefined = SUPPORTED_NETWORK["avalanche_c_chain"];
+    expect(avalancheCChain).toBeDefined();
+    expect(avalancheCChain?.displayName).toBe("Avalanche C-Chain");
+  });
+
+  it("contains Base", () => {
+    const base: Network | undefined = SUPPORTED_NETWORK["base"];
+    expect(base).toBeDefined();
+    expect(base?.displayName).toBe("Base");
+  });
+});
+
+describe("EIP155_GOERLI_CHAINS", () => {
+  it("contains Ethereum Goerli", () => {
+    const ethereumGoerli: Network | undefined = SUPPORTED_NETWORK["ethereum_goerli"];
+    expect(ethereumGoerli).toBeDefined();
+    expect(ethereumGoerli?.displayName).toBe("Ethereum Goerli");
```

Comment:
You can combine both assertions with only `expect(ethereumGoerli?.displayName).toBe("Ethereum Goerli");`
The less we write code, the less we have to maintain or change it.
The second assertion is what you want to test and includes the first one.

Reply:
Yes indeed !